### ISSUE TITLE
Being able to run sandboxes in production mode

### ISSUE THREAD

**Issue by drcmda** [codesandbox/codesandbox-client#2027]
# 🌈 Feature

Quoting Dan Abramov [on Twitter](https://twitter.com/dan_abramov/status/1136527985480126464):

> at least, for React, just switching NODE_ENV is enough. Maybe offer an option?

I wasn't aware that csb isn't running in prod, had no idea that dev mode in general has this kind of impact: https://twitter.com/dan_abramov/status/1136526025393754112

The original demo is here: https://codesandbox.io/s/peaceful-johnson-txtws it goes so slow that it's barely moving at all. 

I am mostly using csb for presentational matters, when you want to demonstrate a library in the best light or some functionality. It would be nice to know the underlying framework's doing its best. 🙂

**Comment by CompuIves** [comment]
Good point! The tricky thing about enabling "production mode" is that we don't do the uglifying/tree-shaking, so we need to communicate clearly that we're only changing the environment variable there and that it doesn't reflect real prod apps.

There are two ways I can think of making this work:

1. We add it as an option to `sandbox.config.json`
2. We fix the sandbox if `NODE_ENV=production` in `.env` file. We do support `.env` files right now, but I added a hardcoded optimization to only include the dev bundle in the case of React. That's why this doesn't work: https://codesandbox.io/s/jovial-jepsen-efe8f. I can remove the optimization in case of a changed env variable, it would also be good to document this better.

**Comment by drcmda** [comment]
Both seem like a good way, thanks for considering this Ives!

**Comment by drcmda** [comment]
still much needed :-)

**Comment by SaraVieira** [comment]
True, removing the label

**Comment by diegohaz** [comment]
Bump

**Comment by oliviertassinari** [comment]
We used to be able to do this by deploying to Netlify or Vercel. It seems that these options were removed.
